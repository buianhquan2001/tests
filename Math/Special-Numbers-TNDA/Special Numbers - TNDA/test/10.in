30
1458 695
4511 3581
3643 4487
1453 769
3581 2536
3675 1358
2634 839
2455 2346
738 2894
4572 3281
794 2231
4700 2336
4587 517
678 4414
4403 2070
2333 1957
4551 1047
2704 4629
3509 3723
1247 1206
3739 3373
3180 2643
2076 3767
1449 2732
1555 4819
4926 3283
4456 1129
1470 763
986 4771
714 3155
