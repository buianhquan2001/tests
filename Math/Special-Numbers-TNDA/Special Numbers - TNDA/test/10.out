668775957
924385969
732541306
461623468
400438673
792480209
879206338
21480204
708260717
30671935
937635081
186320594
738281858
287085092
572017178
716345624
314730877
169092502
420605254
363278890
39528965
161848156
589548355
650200432
105007872
497325937
787988614
376961857
949696378
578206610
